Tangem/Modules/Send/SendModel.swift

```diff
@@ -246,12 +246,12 @@ class SendModel {
             }
             .store(in: &bag)
 
-        Publishers.CombineLatest(validatedAmount, fee)
+        fee
             .removeDuplicates {
                 $0 == $1
             }
```

Comment:
Просто на будущее можно так, если `Element: Equatable`

```suggestion
            .removeDuplicates()
```

Reply:
Сделал. Осталось после CombineLatest